vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentListComponent.java

```diff
@@ -60,68 +62,69 @@ public ExperimentListComponent(
     Objects.requireNonNull(addExperimentToProjectService);
     Objects.requireNonNull(experimentalDesignSearchService);
     this.addExperimentToProjectService = addExperimentToProjectService;
-    this.experimentCreationDialog = new ExperimentCreationDialog(experimentalDesignSearchService);
+    this.experimentInformationService = experimentInformationService;
```

Comment:
What happened in this class?

Reply:
Previously, we propagated the `ExperimentInformationService` to this class but never used it.
Therefore I changed the logic in this class so it only receives the `Context` Information from the Supportcomponent instead of a `Collection<Experiment>` 
The `ExperimentListComponent` then retrieves all the experiment of the provided projectId in the `Context` via the `ExperimentInformationService`
Finally the `ExperimentInformationDialog` replaces the` ProjectCreationDialog` and is not stored as a class variable but built anew every time, similiar to the `ExperimentalVariableDialog`

Reply:
Do we want to fetch all experiments in the component or in the main?

Reply:
I think it makes more sense to fetch the experiments in the list component since only there they are needed